config_vmxenabled.go

```diff
@@ -12,9 +12,9 @@ const (
 // The settings of software which installed in the VMXEnabled image.
 // This setting is used by both "necogcp create-image" and "necogcp neco-test create-image".
 var VMXEnabledArtifacts = setup.ArtifactSet{
-	GoVersion:       "1.22.5",
-	EtcdVersion:     "3.5.14",
-	PlacematVersion: "2.4.6",
+	GoVersion:       "1.23.2",
+	EtcdVersion:     "3.5.17",
+	PlacematVersion: "2.4.7",
 	CoreOSVersion:   "3815.2.5",
```

Comment:
Please Update Flatcar too.
```suggestion
	CoreOSVersion:   "3975.2",
```